app/views/devise/registrations/edit.html.erb

```diff
@@ -0,0 +1,52 @@
+<h2>Edit <%= resource_name.to_s.humanize %></h2>
+
+<%= form_for(resource, as: resource_name, url: registration_path(resource_name), html: { method: :put }) do |f| %>
+  <%= render "devise/shared/error_messages", resource: resource %>
+
+   <div class="field">
+    <%= f.label :name %><br />
+    <%= f.text_field :name %>
+  </div>
+
+  <div class="field">
```

Comment:
### Optional
- [ ] It would be better to display error message when users are unsuccessful in registering or logging in